wfsim/load_resource.py

```diff
@@ -47,24 +43,36 @@ def __init__(self, config={}):
             # Pulse
             self.photon_area_distribution = get_resource(self.config['photon_area_distribution'], fmt='csv')
 
-            # S1
-            self.s1_light_yield_map = itp_map(self.config['s1_light_yield_map'], fmt='json')
-            self.s1_pattern_map = itp_map(self.config['s1_pattern_map'], fmt='json.gz')
-
-            # S2
-            self.s2_light_yield_map = itp_map(self.config['s2_light_yield_map'], fmt='json')
-            if config['experiment']=='XENON1T':
+            if config['detector']== 'XENON1T':
+                # S1
+                self.s1_light_yield_map = itp_map(self.config['s1_light_yield_map'], fmt='json')
+                self.s1_pattern_map = itp_map(self.config['s1_pattern_map'], fmt='json.gz')
+                # S2
+                self.s2_light_yield_map = itp_map(self.config['s2_light_yield_map'], fmt='json')
                 self.s2_per_pmt_params = get_resource(self.config['s2_per_pmt_params'], fmt='csv')
-            if config['experiment']=='XENONnT':
-                self.s2_per_pmt_params = itp_map(self.config['s2_pattern_map'], fmt='json.gz')
-            # Electron After Pulses
-            # self.uniform_to_ele_ap = get_resource(self.config['ele_ap_cdfs'], fmt='npy')
+
+            if config['detector'] == 'XENONnT':
+                self.s1_light_yield =  get_resource('/Users/petergaemers/Downloads/XENONnT_s1_xyz_patterns_corrected_MCv3.0.0.json.gz',fmt = 'json.gz')
```

Comment:
It's strange that lce and pattern are using the same map?

Reply:
I asked Lutz. The LCE map is the sum over all channels in the pattern map. And it's a bit redundunt to have two maps with the same information

Reply:
The one addition that you get from the light yield map is the pe/kev value. If you make the light yield relative it is identical to the relative LCE (but from MC of course, instead of data). So if you anyway using the light yield map just for relative values it's fine ... Otherwise we have to find another approach here ;)